bfauxpwr.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+
+# This feature is supported only on OCP 3.0 cards
+
+powerconf=/etc/acpi/events/powerconf
+powercontrol=/etc/acpi/actions/powercontrol.sh
+systemdconf=/etc/systemd/logind.conf
+
+mkdir -p /etc/acpi/events
+touch $powerconf
+echo \
+"event=button/power.*
+action=$powercontrol
+" > $powerconf
+
+mkdir -p /etc/acpi/actions
+touch $powercontrol
+echo \
+"#!/bin/sh
+
+echo 441 > /sys/class/gpio/export
+value=\$(cat /sys/class/gpio/gpio441/value)
+
+echo \$value > /root/log
+
+for i in {1..7}
```

Comment:
Probably not use hard coded 7? SHould we update for all cores expect 0?

Reply:
Yes, we have to power down all secondary cores